.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -124,5 +124,5 @@ body:
           required: true
         - label: I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
           required: true
-        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).
+        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/mdn-community/discussions).
```

Let's link to the org discussions instead:

```suggestion
        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions).
```